scripts/import_srpm.py

```diff
@@ -2,12 +2,45 @@
 import argparse
 import logging
 import os
+import shutil
 import subprocess
+from glob import glob
+
 
 def call_process(args):
     logging.debug("$ %s", args)
     subprocess.check_call(args)
 
+def pipe_commands(*commands: list[str]) -> bytes:
+    if not commands:
+        raise ValueError("The 'commands' list cannot be empty.")
+    if any(not cmd for cmd in commands):
+        raise ValueError("All commands in the list must be non-empty.")
+
+    processes: list[subprocess.Popen[bytes]] = []
+    last_process_stdout = None
+
+    for i, cmd_args in enumerate(commands):
```

Comment:
I don't see any usages of the variable `i`, why is enumerate used here?

Reply:
I used it, then dropped it while writing the function, but forgot it there.

This is the kind of thing that makes me like static analysis :)